src/Microsoft.AspNetCore.Razor.Design/project.json

```diff
@@ -12,7 +12,10 @@
       "taghelper",
       "aspnetcore",
       "taghelpers"
-    ]
+    ],
+    "files": {
+      "include": [ "buildCrossTargeting/*", "build/*" ]
```

Comment:
Does that work for TFMs that aren't specified by `build/{tfm}`? Aka if a user targets `netcoreapp1.1` wont it fail to import our target which is 1.0?